src/games/kalambury/Board/GameBoard.js

```diff
@@ -23,24 +25,34 @@ const confettiConfig = {
   colors: shuffle(COLORS).slice(0, 5),
 };
 
-const GameBoard = ({
-  envokeLastAnswer,
-  guessInputRef,
-  guess,
-  setGuess,
-  canChangePhrase,
-  ChangePhrase,
-}) => {
-  const { G, ctx, moves, playerID, rawClient } = useBoardGame();
+const GameBoard = ({ guess, setGuess, envokeLastAnswer, guessInputRef }) => {
+  const {
+    G,
+    moves,
+    player: { isDrawing },
+    playerID,
+    rawClient,
+  } = useBoardGame();
   const { t } = useTranslation("kalambury");
   const [lines, setLines] = useState([]);
-  const isDrawing = ctx.activePlayers[playerID] === "draw";
+  const [remainingSeconds, setRemainingSeconds] = useState(G.turnEndTime - currentTime());
   const [lastSuccess, setLastSuccess] = useState(false);
   const lastUserGuess = filterActions(G.actions, playerID, "guess")[0] || {
     id: null,
     success: false,
   };
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      const seconds = G.turnEndTime - currentTime();
+      if ((seconds <= 0 && isDrawing) || seconds < 5) {
```

Comment:
Should it really be `seconds < 5`? This condition will be met before the first one every time

Reply:
no, it should be `-5` Good catch 👍